cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -115,6 +115,7 @@ set(config_module_list
 	modules/ekf2
 	#modules/local_position_estimator
 	#modules/position_estimator_inav
+	#modules/landing_target_estimator
```

Comment:
Try to enable?

Reply:
Still no :(
`region `flash' overflowed by 849 bytes`